app/code/Magento/Theme/view/frontend/web/css/validate.css

```diff
@@ -1,20 +0,0 @@
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-input.mage-error, textarea.mage-error, ul.mage-error, select.mage-error {
-    background: none repeat scroll 0 0 #FAEBE7 !important;
-    border: 1px dashed #EB340A !important;
-
-}
-div.mage-error {
```

Comment:
Looks like it's still used 
https://github.com/magento/magento2/blob/7c6b6365a3c099509d6f6e6c306cb1821910aab0/app/code/Magento/Sales/view/adminhtml/web/order/create/giftmessage.js#L70
https://github.com/magento/magento2/blob/6a0db3b86e0214eacdd1a3b47cca78bede766ca7/app/code/Magento/Checkout/view/frontend/web/js/region-updater.js#L145

Reply:
The same here, looks to be covered by the following mixin `lib/web/css/source/lib/_forms.less:317`. Since the file isn't loaded/called anywhere, we might be sure that the styles aren't used.